### PR TITLE
added axios wrapper

### PR DESCRIPTION
**Pull request by ReginaVarybrus**
In this pull request added axios wrapper, changed file with requests and fixed bug in the RegisterForm component, there are wasn’t route path after authentication to the chat page. Also moved interceptors and function with token set and unset to the component with  axios wrapper.

### CODE REVIEW

File: vite.config.mjs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/// <reference types="vitest" />
```
Comment (DimaShubin) [inline vite.config.mjs]: Why did you rename this file?

File: src/services/api.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = 'https://talk-and-travel.pp.ua/api/authentication';
+
+axios.interceptors.request.use(request => {
+  console.log('Starting Request', request);
```
Comment (DimaShubin) [inline src/services/api.js]: delete it

File: src/services/api.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return request;
+});
+
+axios.interceptors.response.use(response => {
+  console.log('Response:', response);
+  return response;
+});
+
+export const token = {
+  set(token) {
```
Comment (DimaShubin) [inline src/services/api.js]: "token" should be renamed here. Seems like you still haven't configure your editor properly to make esLisnt and Prettier work.

File: src/redux-store/AuthOperations/AuthOperations.js [hunk 4]
```diff
@@ -68,13 +54,19 @@ export const logIn = createAsyncThunk('auth/login', async userData => {
...
+
     token.unset();
-  } catch (e) {
-    console.log(e.message);
+  } catch (error) {
+    throw new Error(error.message);
   }
 });
 
+// TODO fetchCurrentUser, updateUser and other requests shuold be update when we'll take the Profile component in dev
```
Comment (DimaShubin) [inline src/redux-store/AuthOperations/AuthOperations.js]: Whu didn't you apply changes for all requests?

**Review by DimaShubin — CHANGES_REQUESTED** [review]

File: src/redux-store/AuthOperations/AuthOperations.js (hunk 4 above)
Comment (ReginaVarybrus) [reply src/redux-store/AuthOperations/AuthOperations.js]: Because these components don't work still and I don't know how these requests will be working in the future

File: src/services/api.js (hunk 3 above)
Comment (ReginaVarybrus) [reply src/services/api.js]: Done

File: src/services/api.js (hunk 2 above)
Comment (ReginaVarybrus) [reply src/services/api.js]: Should I delete only iterseptors?

File: vite.config.mjs (hunk 1 above)
Comment (ReginaVarybrus) [reply vite.config.mjs]: Because of this advice in the terminal 'The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.' I renamed it back

File: src/redux-store/AuthOperations/AuthOperations.js (hunk 4 above)
Comment (ReginaVarybrus) [reply src/redux-store/AuthOperations/AuthOperations.js]: Done, but sendDataCountryToBackend request is still under TODO comment

File: src/services/api.js (hunk 2 above)
Comment (DimaShubin) [reply src/services/api.js]: Those are do nothing actually right now so let's delete them.

Comment (ReginaVarybrus) [reply src/services/api.js]: Done

File: src/redux-store/AuthOperations/authSlice.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
+import { createSlice } from '@reduxjs/toolkit';
```
Comment (DimaShubin) [inline src/redux-store/AuthOperations/authSlice.js]: Why did you move this file?

File: src/services/api.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
```
Comment (DimaShubin) [inline src/services/api.js:2]: Something like this.
